lib/index.js

```diff
@@ -31,6 +34,14 @@ function runClient () {
   })
 
   var cm = new ChainManager(config, nm, bc) // eslint-disable-line
+
+  if (config.rpc) {
+    const { rpcport, rpcaddr } = config
+    const manager = new RPCManager(cm.getBlockchain(), config)
+    const server = jayson.server(manager.getMethods())
+    config.logger.info(`HTTP endpoint opened: http://${rpcaddr}:${rpcport}`)
+    server.http().listen(rpcport)
```

Comment:
Was a bit sceptical about the naming but wasn't aware that ``consts`` in an ``if`` clause remain local not trickling in the global space (tested this), so this should be ok.